citar-org.el

```diff
@@ -83,6 +77,19 @@ Each function takes one argument, a citation."
   :group 'citar-org
   :type '(repeat function))
 
+(defcustom citar-org-note-setup-id 'org-roam-only
+  "Whether `citar-org-note-setup` adds an Org ID."
+  :group 'citar-org
+  :type '(choice (const :tag "Always" t)
+                 (const :tag "Only in Org-roam buffers" org-roam-only)
+                 (const :tag "Never" nil)))
+
+(defcustom citar-org-note-setup-roam-ref t
+  "Whether `citar-org-note-setup` adds an Org-roam reference."
+  :group 'citar-org
+  :type '(choice (const :tag "In Org-roam buffers" t)
+                 (const :tag "Never" nil)))
+
```

Comment:
Are these really needed?

If you're using org-roam v2, these (at least the ROAM_REF) aren't really optional, so it seems can just add it if in an org-roam buffer?

Id is a little different though, since it's not specific to org-roam.

Reply:
Yeah, that makes sense. I can remove them and implement the behaviour you describe (always add ID, and in Org-roam also add roam_ref). If somebody doesn't even want the ID, then they can just remove that function from the skeleton.